folly/ExceptionWrapper-inl.h

```diff
@@ -657,17 +657,15 @@ inline bool exception_wrapper::with_exception(Fn fn) const {
 
 template <class... CatchFns>
 inline void exception_wrapper::handle(CatchFns... fns) {
-  using AllStdEx =
-      exception_wrapper_detail::AllOf<IsStdException, arg_type<CatchFns>...>;
+  using AllStdEx = StrictConjunctionBy<IsStdException, arg_type<CatchFns>...>;
```

Comment:
Hmm, so thinking again, this could just be

```
using AllStdEx = StrictConjunction<IsStdException<arg_type<CatchFns>>...>;
```

Reply:
Good spot! I agree. Just changed it and the other call site in this file.